doc/en/doctest.rst

```diff
@@ -171,20 +212,14 @@ Note that like the normal ``conftest.py``, the fixtures are discovered in the di
 Meaning that if you put your doctest with your source code, the relevant conftest.py needs to be in the same directory tree.
 Fixtures will not be discovered in a sibling directory tree!
 
-Output format
--------------
-
-.. versionadded:: 3.0
+Skipping tests dynamically
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can change the diff output format on failure for your doctests
-by using one of standard doctest modules format in options
-(see :data:`python:doctest.REPORT_UDIFF`, :data:`python:doctest.REPORT_CDIFF`,
-:data:`python:doctest.REPORT_NDIFF`, :data:`python:doctest.REPORT_ONLY_FIRST_FAILURE`):
+.. versionadded:: 4.4
 
-.. code-block:: bash
+You can use ``pytest.skip`` to dynamically skip doctests. For example::
 
-    pytest --doctest-modules --doctest-report none
-    pytest --doctest-modules --doctest-report udiff
-    pytest --doctest-modules --doctest-report cdiff
-    pytest --doctest-modules --doctest-report ndiff
-    pytest --doctest-modules --doctest-report only_first_failure
+    >>> import sys, pytest
+    >>> if sys.platform.startswith('win'):
+    ...     pytest.skip('this doctest does not work on Windows')
+    ...
```

Comment:
Right, but this is ugly to include in documentation; my approach with Sybil has been to always make the documentation the key and just work around whatever's needed to check the examples it contains are correct (hence [capture](https://sybil.readthedocs.io/en/latest/parsers.html#capture), [skip](https://sybil.readthedocs.io/en/latest/parsers.html#skip) and [invisible-code-block](https://sybil.readthedocs.io/en/latest/parsers.html#codeblock)!).

Reply:
Oh I agree it is not ideal, it certainly can be improved! 👍

Reply:
Yes, use Sybil and hide everything that the user doesn't need to see ;-)